aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/AeroGearGCMPushRegistrar.java

```diff
@@ -159,6 +168,11 @@ protected Exception doInBackground(Void... params) {
                         postData.addProperty("variantId", variantId);
                         postData.addProperty("secret", secret);
                         presistPostInformation(context.getApplicationContext(), postData);
+                        GcmPubSub gcmPubSub = gcmPubProvider.get(context);
+                    
+                        for (String catgory : categories) {
```

Comment:
Awesome, I've recorded that in a sub task : https://issues.jboss.org/browse/AGDROID-539